cesium/open.go

```diff
@@ -42,12 +42,14 @@ func Open(dirname string, opts ...Option) (*DB, error) {
 		shutdown:   signal.NewShutdown(sCtx, cancel),
 	}
 	for _, i := range info {
-		key := core.ChannelKey(lo.Must(strconv.Atoi(i.Name())))
-		if err != nil {
-			return nil, err
-		}
 		if i.IsDir() {
-			if err = _db.openVirtualOrUnary(Channel{Key: key}); err != nil {
+			key, err := strconv.Atoi(i.Name())
+			if err != nil {
+				_db.options.L.Error("failed parsing existing folder to channel key", zap.Error(err))
```

Comment:
I don't think there's any point to call this variable `_db` anymore. Let's just rename to `db`